docs/references/ic-interface-spec.md

```diff
@@ -2293,6 +2295,26 @@ Indicates various information about the canister. It contains:
 
 Only the controllers of the canister or the canister itself can request its status.
 
+#### Memory Metrics {#ic-canister_status-memory_metrics}
+
+    * `heap_size`: Represents the Wasm memory usage of the canister, i.e. the heap memory used by the canister's WebAssembly code.
+
+    * `stable_size`: Represents the stable memory usage of the canister, which is the memory that persists across canister upgrades.
```

Comment:
wasm heap (or main) memory is also persisted for Motoko EOP

Reply:
This is true. However, it would feel kinda weird to mention some special case for Motoko OP in the spec. What would you suggest? Maybe just drop the part "which is the memory that persists across canister upgrades" for stable memory? I guess we can argue that stable memory is sufficiently described in other sections of the spec and therefore here we do not need to overdo it.

Reply:
Indeed, I'd drop `which is the memory that persists across canister upgrades.`